src/utils/helpers/list/useGetFilterDefinitions.ts

```diff
@@ -197,3 +225,32 @@ const getDateRangeForUnit: GetDateRangeForUnit = (unit, value = 0) => {
       .startOf('day'),
   ]
 }
+
+const checkFilterVisibility = (filterKey: string) => {
+  let filtersAreVisibleTo: USER_ROLES[] = []
+  
+  const visibleToApplicant = [
+    LIST_FILTERS.LAST_ACTIVE_DATE,
+    LIST_FILTERS.DEADLINE_DATE,
+    LIST_FILTERS.STATUS,
+    LIST_FILTERS.OUTCOME,
+    LIST_FILTERS.APPLICANT,
+    LIST_FILTERS.APPLICANT_ACTION,
+  ]
+  if (visibleToApplicant.find((visibleFilter) => visibleFilter === filterKey)) filtersAreVisibleTo.push(USER_ROLES.APPLICANT)
+  
+  const visibleToReviewer: LIST_FILTERS[] = [
+    LIST_FILTERS.LAST_ACTIVE_DATE,
+    LIST_FILTERS.DEADLINE_DATE,
+    LIST_FILTERS.STATUS,
+    LIST_FILTERS.OUTCOME,
+    LIST_FILTERS.STAGE,
+    LIST_FILTERS.ORGANISATION,
+    LIST_FILTERS.APPLICANT,
+    LIST_FILTERS.ASSIGNERS,
+    LIST_FILTERS.REVIEWERS,
+    LIST_FILTERS.REVIEWER_ACTION,
+  ]
+  if (visibleToReviewer.find((visibleFilter) => visibleFilter === filterKey)) filtersAreVisibleTo.push(USER_ROLES.REVIEWER)
+  return filtersAreVisibleTo
```

Comment:
This seems like a lot of unnecessary bloat when they could all be as simple as the "search" filter above:
```
visibleTo: [USER_ROLES.APPLICANT, USER_ROLES.REVIEWER],
```

Why do you think we need all this?

Reply:
Sorry, not sure what is your suggestion on this one : (

Reply:
It's the same suggestion as above -- you don't need "checkFilterVisiblity" at all.